go/common/rpc/generated/enclave.pb.go

```diff
@@ -271,6 +271,99 @@ func (x *SystemError) GetErrorString() string {
 	return ""
 }
 
+type GetTotalContractCountRequest struct {
+	state         protoimpl.MessageState
+	sizeCache     protoimpl.SizeCache
+	unknownFields protoimpl.UnknownFields
+}
+
+func (x *GetTotalContractCountRequest) Reset() {
+	*x = GetTotalContractCountRequest{}
+	if protoimpl.UnsafeEnabled {
+		mi := &file_enclave_proto_msgTypes[5]
+		ms := protoimpl.X.MessageStateOf(protoimpl.Pointer(x))
+		ms.StoreMessageInfo(mi)
+	}
+}
+
+func (x *GetTotalContractCountRequest) String() string {
+	return protoimpl.X.MessageStringOf(x)
+}
+
+func (*GetTotalContractCountRequest) ProtoMessage() {}
+
+func (x *GetTotalContractCountRequest) ProtoReflect() protoreflect.Message {
+	mi := &file_enclave_proto_msgTypes[5]
+	if protoimpl.UnsafeEnabled && x != nil {
+		ms := protoimpl.X.MessageStateOf(protoimpl.Pointer(x))
+		if ms.LoadMessageInfo() == nil {
+			ms.StoreMessageInfo(mi)
+		}
+		return ms
+	}
+	return mi.MessageOf(x)
+}
+
+// Deprecated: Use GetTotalContractCountRequest.ProtoReflect.Descriptor instead.
+func (*GetTotalContractCountRequest) Descriptor() ([]byte, []int) {
+	return file_enclave_proto_rawDescGZIP(), []int{5}
+}
+
+type GetTotalContractCountResponse struct {
+	state         protoimpl.MessageState
+	sizeCache     protoimpl.SizeCache
+	unknownFields protoimpl.UnknownFields
+
+	Count       int64        `protobuf:"varint,1,opt,name=count,proto3" json:"count,omitempty"`
+	SystemError *SystemError `protobuf:"bytes,2,opt,name=systemError,proto3" json:"systemError,omitempty"`
+}
+
+func (x *GetTotalContractCountResponse) Reset() {
+	*x = GetTotalContractCountResponse{}
+	if protoimpl.UnsafeEnabled {
+		mi := &file_enclave_proto_msgTypes[6]
+		ms := protoimpl.X.MessageStateOf(protoimpl.Pointer(x))
+		ms.StoreMessageInfo(mi)
+	}
+}
+
+func (x *GetTotalContractCountResponse) String() string {
+	return protoimpl.X.MessageStringOf(x)
+}
+
+func (*GetTotalContractCountResponse) ProtoMessage() {}
+
+func (x *GetTotalContractCountResponse) ProtoReflect() protoreflect.Message {
+	mi := &file_enclave_proto_msgTypes[6]
+	if protoimpl.UnsafeEnabled && x != nil {
+		ms := protoimpl.X.MessageStateOf(protoimpl.Pointer(x))
+		if ms.LoadMessageInfo() == nil {
+			ms.StoreMessageInfo(mi)
+		}
+		return ms
+	}
+	return mi.MessageOf(x)
+}
+
+// Deprecated: Use GetTotalContractCountResponse.ProtoReflect.Descriptor instead.
+func (*GetTotalContractCountResponse) Descriptor() ([]byte, []int) {
+	return file_enclave_proto_rawDescGZIP(), []int{6}
```

Comment:
Maybe add constants for indexes? (ignore this comment if it doesn't make sense here)

Reply:
Yeah this file gets generated based on the `enclave.proto` thing so not much we can do about the filth contained within haha. Other than trying not to peek.